notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-756_analysis.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>I suggest renaming the Jira link to point to the new <a href="https://rubinobs.atlassian.net" target="_blank">https://rubinobs.atlassian.net/</a> cloud</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>